package-res/settings.xml

```diff
@@ -12,5 +12,13 @@
     <auto-submit>false</auto-submit>
     <auto-submit-default>false</auto-submit-default>
 	-->
+
+    <!-- The maximum number of rows that will be rendered in a report viewer. A zero value means no limit. -->
+    <maximum-query-limit>100000</maximum-query-limit>
```

Comment:
The row-limits should be disabled by default so that we are compatible with the old versions. Admins can enable row-limits if they want to. 

A user-limit of 500 sounds unreasonable low and now what the case described. The case requests that the row-limit attribute of the report is used for a default value instead.